gloo/docs/installation/install_glooctl.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Install Gloo CLI glooctl
+weight: 1
+---
+
```

Comment:
This page looks good. We should have a "what's next" section or at least links to installing Gloo in this page so it's easy to keep going once they get the cli installed.

Reply:
Or maybe have links to "Kubernetes prep" AND "install gloo" from this page since user could go either way